datascroller/scroller.py

```diff
@@ -417,14 +398,26 @@ def move_highlight_down(self):
     def move_highlight_up(self):
         self.highlight_row -= 1
 
-# TODO(johncmerfeld): this should respond dynamically to config file
 
+def add_help_string(screen, cols):
+    screen.addstr(0, 0, help.BANNER, curses.A_BOLD)
+    screen.chgat(0, 0, cols, curses.A_UNDERLINE)
+    for i, option in enumerate(help.MENU_OPTIONS):
+        exec("screen.addstr(i + 1, 1, help." + option + ", curses.A_BOLD)")
+        exec("screen.addstr(i + 1, cols - 1 - len(help." + option + "_TEXT), help." + option + "_TEXT)")
 
-def get_help_string():
-    help_string = ('Ver: j/k \t Hor: h/l \t Page Down/Up: ctrl+f/+b\t Quit: q\t Help: \'\n' +
-                   'Goto line: ;\t Filter: .\t Query: /\t Exit query/filter: b\t Highlight mode: ,')
 
-    return help_string
+def show_help_view(screen, cols, rows):
+    # TODO(johncmerfeld): too many constants in here
+    width = 40
+    height = 13
+    box1 = screen.subwin(height, width, 1, cols - width)
+    box2 = screen.subwin(height - 2, width - 2, 2, cols - width + 1)
```

Comment:
This looks amazing as it is. I can read everything in the help menu, but the right border of the box is not displaying, even though I can see from the code that the padding is the same. I remember having some issues with the last few columns acting weird. It might be worth trying a bit more padding on the width.

Reply:
Looking into it!

Reply:
Hm, I can see the border. Can you post a screenshot?

Reply:
Yep, I can see the border now too. Tried and tried to recreate the issue but couldn't. There was something going on with my Ubuntu window because the tmux date was also getting cutoff to the right. Since I can't reproduce, I think it's fine as is.

Reply:
Okay, cool. If it comes up again we can always shift the box.